modules/client/examples/01-client/13-prepare-installation.ts

```diff
@@ -5,18 +5,18 @@ title: PrepareInstallation
 ### Prepare installation of a plugin
 */
 
+import { Client } from "@aragon/sdk-client";
 import {
-  Client,
   PrepareInstallationParams,
   PrepareInstallationStep,
-} from "@aragon/sdk-client";
+} from "@aragon/sdk-client-common";
 import { context } from "../index";
 
 // Instantiate the general purpose client from the Aragon OSx SDK context.
 const client: Client = new Client(context);
 
 // Obtained from the plugin metadata
```

Comment:
Where can this plugin metadata be found? Would be great to provide some guidance here

Reply:
Potentially we add it on the top in the description? I do think it's important information on this lives in this file so it's quicker for devs to retrieve the missing info. At least a link to that additional file you mention would do.

Reply:
There's now a link to https://github.com/aragon/simple-storage-example-plugin/blob/main/contracts/release1/build3/build-metadata-R1B3.json#L7-L20